### PR TITLE
WA Group: Sending message to collection

### PR DESCRIPTION
**Pull request by AkhileshNegi**
## Summary
 Target issue is #3434  
 Explain the **motivation** for making this change. What existing problem does the pull request solve?

## Checklist
  Before submitting a pull request, please ensure that you mark these task.

- [ ] Ran `mix setup` in the repository root and test.
- [ ] If you've fixed a bug or added code that is tested and has test cases. 
- [ ] In the case of adding a new API, you added a **postman** request for that.
- [ ] Coding standard and conventions are followed. https://docs.google.com/document/d/1rfU33IjS-ioiIH0TBWpxoLsdyyYdI1tDrqr5HCRIb98/edit?usp=sharing

## Notes
 Please add here if any other information is required for the reviewer.

### CODE REVIEW

File: lib/glific/providers/maytapi/message.ex [hunk 1]
```diff
@@ -37,6 +40,73 @@ defmodule Glific.Providers.Maytapi.Message do
     })
   end
 
+  @doc false
```
Comment (madclaws) [inline lib/glific/providers/maytapi/message.ex:43]: docs missing

Comment (madclaws) [inline lib/glific/providers/maytapi/message.ex]: docs missing

File: lib/glific/providers/maytapi/message.ex [hunk 2]
```diff
@@ -37,6 +40,73 @@ defmodule Glific.Providers.Maytapi.Message do
...
+  @spec send_message_to_wa_group_collection(Group.t(), map()) ::
+          {:ok, map()}
+  def send_message_to_wa_group_collection(group, attrs) do
+    wa_group_collections =
+      WaGroupsCollections.list_wa_groups_collection(%{
+        filter: %{group_id: group.id, organization_id: group.organization_id}
+      })
+      |> Repo.preload([:wa_group])
+
+    create_wa_group_message(wa_group_collections, group, attrs)
```
Comment (madclaws) [inline lib/glific/providers/maytapi/message.ex:56]: Why do we create wa_message twice?, here and at 66th line.

File: lib/glific/providers/maytapi/message.ex (hunk 1 above)
Comment (AkhileshNegi) [reply lib/glific/providers/maytapi/message.ex]: will work on that, but amisha created the entire file with no doc so I asked her to update other as well

File: lib/glific/providers/maytapi/message.ex (hunk 2 above)
Comment (AkhileshNegi) [reply lib/glific/providers/maytapi/message.ex:56]: One is group message that is shown in collection screen and other is the actual msg. We can discuss more over call

**Pull request merged**